### PR TITLE
Update to alpine 3.11 golang 1.14.1 librdkafka 1.4.0

### PR DESCRIPTION
**Pull request by minitauros**
1.14 has been around for a while now. Slowly time to think about upgrading at some point?

---

Images can be built successfully but have not been tested. They have been pushed, so feel free to test with them.

### CODE REVIEW

File: alpine-golang-librdkafka/alpine3.11-golang1.14.1-librdkafka1.4.0-static/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,41 @@
+FROM golang:1.14.1-alpine3.11
```
Comment (si-m) [inline alpine-golang-librdkafka/alpine3.11-golang1.14.1-librdkafka1.4.0-static/Dockerfile:1]: The previous time the 3.11 with the static libraries that the librdkakfa need didn't work. Is now okay?

Comment (minitauros) [reply alpine-golang-librdkafka/alpine3.11-golang1.14.1-librdkafka1.4.0-static/Dockerfile:1]: I don't know, I haven't tested it yet. I opened this PR and invited you guys to slowly start considering upgrading Go version. But also feel free to stay with 1.13 for a while more, it runs perfectly fine and there is no urgent need to upgrade. I'll start testing with these images to see how it goes.

**Comment by Slijkhuis** [comment]
I'll try to keep it mind to try this image when working on a project that uses librdkafka.

Re-invite me for review when you have a bit more results on whether this actually works.

**Comment by minitauros** [comment]
Closing in favor of https://github.com/unitedwardrobe/golang-librdkafka/pull/9.

**Pull request closed without merging**